styles.css

```diff
@@ -308,24 +331,51 @@ body {
 
   .main {
     padding: 20px 45% 100px 45px;
-    background: top 5px right 49px no-repeat url(img/img.svg);
+    background: top 4px right 9% no-repeat url(img/img.svg);
   }
 
   .main__title {
-    margin: 0 0 35px 0;
+    margin: 0 0 32px 0;
     padding-bottom: 5px;
   }
 
   .main__description {
-    max-width: 67%;
+    max-width: 69%;
+    margin-bottom: 43px;
   }
 
   .main__button {
     padding: 13px 21px;
   }
 }
 
-@media (max-width: 377px) {
+@media (min-width: 760px) and (max-width: 1000px) {
+  .wrapper {
+    width: 95%;
+    margin: auto;
+  }
+
+  .menu {
+    margin-left: 15%;
+    margin-right: 5%;
+  }
+
+  .header {
+    margin-bottom: 40px;
+  }
+
+  .main {
+    background: top 4px right 9%/55% no-repeat url(img/img.svg);
+  }
+
+  .main__title {
+    font: bold 32px/42px 'pt_serif', arial, helvetica, sans-serif;
+    max-width: 70%;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6378545)